Developers/Writing_Bears.rst

```diff
@@ -150,6 +150,42 @@ This will show the user a bunch of information related to the bear like:
 - A description of what the bear does - The sections which uses it - The
 settings it uses (optional and required)
 
+.. note::
+
+    The bears are not yet installed. We still have to specify
+    the bear directory using ``-d`` or ``--bear-dirs`` flag.
+
+
+Installing the New coala-bears Locally
```

Comment:
i dont think you understood the issue correctly. it implies (or at least from what i understand) that you should first install coala-bears to test a locally written bear.